tests/test_cli.py

```diff
@@ -44,6 +44,11 @@
     ('path', DOCKERFILE_OK_PATH),
 ])
 
+
+def get_uuid():
```

Comment:
Can we turn this into a proper fixture in conftest.py?
```python
@pytest.fixture()      
def uuid_value():                                                    
    return uuid.uuid4().hex                                        
                                                                   
                                                                   
@pytest.fixture()                                                  
def temp_image_name(uuid_value):                                             
    return ImageName(repo=("atomic-reactor-tests-%s" % uuid_value))
```

That way we don't need to redefine it here.

Reply:
Since we are only using it in the other fixture and in a setup_method, maybe we should either move it to `tests/util.py` or just go ahead and make the calls to uuid directly instead of having a function?

Reply:
WFM

Reply:
Done